scripts/partition_manager.py

```diff
@@ -440,17 +440,23 @@ def get_required_offset(align, start, size, move_up):
     end = start + size
     align_start = 'start' in align
 
-    if (align_start and start % align['start'] == 0) or (not align_start and end % align['end'] == 0):
-        return 0
+    try:
+        if (align_start and start % align['start'] == 0) or (not align_start and end % align['end'] == 0):
+            return 0
 
-    if move_up:
-        return align['start'] - (start % align['start']) if align_start else align['end'] - (end % align['end'])
-    else:
-        if align_start:
-            return start % align['start']
+        if move_up:
+            return align['start'] - (start % align['start']) if align_start else align['end'] - (end % align['end'])
         else:
-            # Special handling is needed if start is 0 since this partition can not be moved down
-            return end % align['end'] if start != 0 else align['end'] - (end % align['end'])
+            if align_start:
+                return start % align['start']
+            else:
+                # Special handling is needed if start is 0 since this partition can not be moved down
+                return end % align['end'] if start != 0 else align['end'] - (end % align['end'])
+    except TypeError as err:
+        keyword = 'start' if align_start else 'end'
+        message = (str(err.args[0]) + f"\n{' '*len('TypeError: ')}elements in align: {{{keyword}:{align[keyword]}}} is not type of \'int\'\n", )
```

Comment:
I couldn't find the pythonic way of doing this. Changed it so that it doesn't modify internal variables such as err.args or accessing private "__" variables.
`raise TypeError(msg) from err` so that you can see the cause and get the error message.